src/plugins/irc-events/message.js

```diff
@@ -10,6 +10,11 @@ module.exports = function(irc, network) {
 			return;
 		}
 
+		if (data.message.substring(0,4) == '?OTR') {
+			// Hide OTR messages
+			return;
+		}
```

Comment:
Hey look, `if (data.message.substring(0,4) == '?OTR') return;` is less than 80 characters! :D

I think I would even prefer something along the lines of `if (this.otrStore.isOtr(data.message))`, what do you think?